RELEASE.md

```diff
@@ -1,3 +1,15 @@
+# Release 2.2.2
```

Comment:
As you suggested offline, the Release notes are not yet complete.

Reply:
I expanded the notes in RELEASE.md a little. PTAL.

Reply:
LGTM